lips/lip-10.md

```diff
@@ -0,0 +1,221 @@
+---
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 11/03/2020
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
+
+# Motivation
+Libra ID provides a convenient way to identify a user's account within a VASP. There are two main use cases for Libra ID:
+* For peer-2-peer payments, Libra ID allows a user to provide a human-readable ‘identifier’ to the sender. Basically, instead of sending a [lip-5 address](https://github.com/libra/lip/blob/master/lips/lip-5.md) (that needs to be refreshed for each payment, ideally), the receiver can just share their Libra ID once. In this case Libra ID plays the role of an email address for payments. Benefits for this use case:
+  * Libra ID is a user-friendly, human-readable identifier, unlike lip-5 addresses
+  * The Libra ID resolution process can facilitate the exchange of rich information(e.g. nicknames, profile picture, etc.) about both the sender and receiver of the payment
+  * Libra ID provides better privacy compared to using sub-addresses, because Libra ID is not shared on the chain and will always use a single-use identifier for the on-chain ID
```

Comment:
This is not always true.
Example is this - you and friends went to restaurant and you paid for everyone. Now you have a group chat in <messenger of your choice> and want to share your Libra wallet address so that they can pay you back.

There is no convenient way to share lip-5 address so that participants  can only send money, but not track payments from others. Of course in theory you can create subaddress for each, but this is not practical